llvm/lib/ObjCopy/COFF/COFFObjcopy.cpp

```diff
@@ -316,6 +316,12 @@ static Error handleArgs(const CommonConfig &Config,
       Obj.PeHeader.MinorSubsystemVersion = *COFFConfig.MinorSubsystemVersion;
   }
 
+  if (!Config.AdjustSectionVMA.empty()) {
```

Comment:
This isn't the correct place to handle unsupported options. Please see ConfigManager.cpp for how to handle options that are not yet implemented for some platforms, but might be in the future.